test/replcompletions.jl

```diff
@@ -45,6 +46,7 @@ function temp_pkg_dir(fn::Function)
     fn()
   finally
     rm(tmpdir, recursive=true)
+    ENV = environment
```

Comment:
This doesn't seem right - at the REPL this kind of thing gives a `WARNING: imported binding for ENV overwritten in module Main`. There's a `withenv` function that would probably be better for this.